uvls/src/main.rs

```diff
@@ -544,11 +557,60 @@ impl LanguageServer for Backend {
                         arguments: Some(vec![uri_json]),
                     }),
                     data: None,
-                }
+                },
             ]))
         }
     }
 
+    async fn code_action(&self, params: CodeActionParams) -> Result<Option<CodeActionResponse>> {
+        for diagnostic in params.context.diagnostics {
+            // Checks if there is a quick fix for the current diagnostic message
+            match diagnostic.message.as_str() {
+                "name contains a dash (-)" => {
```

Comment:
Is there some other way to push an error (e.g. with an enum of some sort) to make it more traceable.  
A string by itself might be easy to change / produce an error in the future as soon as someone edits the error message somehow.  
Otherwise looks good